### ISSUE TITLE
Why inputs and labels are same while training?

### ISSUE THREAD

**Issue by vaishalishrivastava** [RuolinZheng08/twewy-discord-chatbot#12]
```
for step, batch in enumerate(epoch_iterator):

            # Skip past any already trained steps if resuming training
            if steps_trained_in_current_epoch > 0:
                steps_trained_in_current_epoch -= 1
                continue

            inputs, labels = (batch, batch)
            if inputs.shape[1] > 1024: continue
            inputs = inputs.to(args.device)
            labels = labels.to(args.device)
            model.train()
            outputs = model(inputs, labels=labels)
```

**Comment by whiteeat** [comment]
Yeah, I spot that too. I also observe other repos of training chat bot based on gpt2 and they do the similar thing. I really doubt AI can learn the style of speech from a specific character using this method of training.
